contracts/stellar-interchain-token-service/src/tests/register_canonical_token.rs

```diff
@@ -14,22 +14,19 @@ use crate::types::TokenManagerType;
 fn register_canonical_token_succeeds() {
     let (env, client, _, _, _) = setup_env();
     let token_address = Address::generate(&env);
-    let registered_token = &env.register_stellar_asset_contract_v2(token_address);
-    let expected_id = client.canonical_interchain_token_id(&registered_token.address());
+    let token = &env.register_stellar_asset_contract_v2(token_address);
```

Comment:
The argument to this function still doesn't make sense. See the doc and our existing usage
https://docs.rs/soroban-sdk/latest/soroban_sdk/struct.Env.html#method.register_stellar_asset_contract_v2